packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -23,142 +23,228 @@ describe('meeting', () => {
   });
 
   describe('toggleIncomingClientAudio', () => {
+    const allowedContexts = [FrameContexts.sidePanel, FrameContexts.meetingStage];
     it('should not allow calls before initialization', () => {
       expect(() => meeting.toggleIncomingClientAudio()).toThrowError('The library has not yet been initialized');
     });
 
-    it('should successfully toggle the incoming client audio', async () => {
-      await desktopPlatformMock.initializeWithContext('sidePanel');
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
+      }
```

Comment:
possibly stupid question, why do we need an if statement

Reply:
if I do the return statement, I see options to debug/Run for the test, but if I nest the test inside for loop, I don't see the debug/run option. Let me search if there is any other better option online
![image](https://user-images.githubusercontent.com/88564543/154369362-d13bef60-93d5-49fd-b977-1fc9b7c2cd68.png)
![image](https://user-images.githubusercontent.com/88564543/154369062-b87b468c-1a8a-4ad7-9188-cf9922193bae.png)

Reply:
@Jessica-Koch because we want an API to throw error if we initialize with a context in which capability is not supported in (and works fine in other cases).